website/docs/error-reference.md

```diff
@@ -1459,6 +1928,136 @@ class ChildTwo < T::Struct
 end
 ```
 
+## 5042
+
+Sorbet does not allow defining a `private` method in an `interface!`. Note that
+this limitation is lifted if the `module` is declared `abstract!` not
+`interface`.
+
+The justification for why this is an error has been lost to the sands of time.
+It seems reasonable to implement support for this.
+
+<https://github.com/sorbet/sorbet/issues/5687>
+
+## 5043
+
+The syntax for declaring type aliases has changed. What used to be written like
+this:
+
+```ruby
+# BAD
+X = T.type_alias(Integer)
+```
+
+must now be written like this, with a block argument instead of a positional
+argument:
+
+```ruby
+# GOOD
+X = T.type_alias {Integer}
+```
+
+The new syntax avoids incurring eagerly evaluating the right-hand side of the
+type alias, potentially forcing one or more constants to be autoloaded before
+they would otherwise be required. This improves load-time performance in
+lazily-loaded environments (usually: development environments) and also prevents
+certain modes of use of Sorbet to introduce load-time cyclic references.
```

Comment:
```suggestion
certain modes of the use of Sorbet to introduce load-time cyclic references.
```

I'm not sure what the right fix here is, but "modes of use of Sorbet" sounds kind of weird to me.

Reply:
committed an alternative phrasing.